prov/rxm/src/rxm_ep.c

```diff
@@ -129,7 +121,6 @@ static int rxm_buf_pool_create(int local_mr, size_t chunk_count, size_t size,
 		FI_WARN(&rxm_prov, FI_LOG_EP_CTRL, "Unable to create buf pool\n");
 		return -FI_ENOMEM;
 	}
-	dlist_init(&pool->buf_list);
 	fastlock_init(&pool->lock);
 	return 0;
 }
```

Comment:
I think this patch will cause an assertion in util_buf_pool_destroy. It checks if all buffers that were alloc'ed were released.

Reply:
Yes, you're right. Sorry, I forgot to check debug version. The RX pool is causing an assertion in the util_buf_pool_destroy() due to posted receives. I'm going to handle this.
Thanks!